tools/performance/PowerShellPerf.wprp

```diff
@@ -0,0 +1,148 @@
+<?xml version='1.0' encoding='utf-8' standalone='yes'?>
+<WindowsPerformanceRecorder
+    Author="PowerShell Performance"
+    Comments="PowerShell Performance"
+    Company="Microsoft Corporation"
+    Copyright="Microsoft Corporation"
+    Team="PowerShell Performance"
+    Version="1.0"
+    >
+  <Profiles>
+    <SystemCollector Id="SystemCollector" Name="NT Kernel Logger">
+      <!-- 1MB * 100 buffers = 100 MB -->
+      <BufferSize Value="1024"/>
+      <Buffers Value="100"/>
+    </SystemCollector>
+
+    <EventCollector Id="EventCollector" Name="EventCollector">
+      <BufferSize Value="1024" />
+      <Buffers Value="100" />
```

Comment:
Your other profiles capture a small subset of the events this one does, but use a much larger set of buffers.

Reply:
This is the profile we care about. I don't recall seeing dropped events, so maybe this is fine?